src/utils/clients/lastfm/formatters/album-search-response.formatter.ts

```diff
@@ -0,0 +1,33 @@
+import get from 'lodash/get';
+
+interface AlbumResult {
+  name: string;
+  artist: string;
+  mbid: string;
+  url: string;
+  image: { '#text': string }[];
+}
+
+export function albumSearchFormatter(response?: unknown) {
+  const albums = [];
+  const rawAlbumList = get(response, 'data.results.albummatches.album', []) as AlbumResult[];
+
+  for (const album of rawAlbumList) {
+    if (album.name === '(null)') continue;
+    albums.push({
+      artist: album.artist,
+      mbid: album.mbid,
+      name: album.name,
+      url: album.url, // <--
```

Comment:
Feel free to remove this comment if you want, 3 years later I have no idea why I added it 🙃

Reply:
Fixed in 8a2863f